### PR TITLE
Issue #1206 - Implementation of PioneerAVR multi-zone control.

### PR DESCRIPTION
**Pull request by Stratehm**
Implementation of PioneerAVR multi-zone control.

Signed-off-by: Antoine Besnard stratehm@hotmail.com

### CODE REVIEW

File: addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -37,14 +34,12 @@
...
-			<channel id="power" typeId="powerChannel" />
-			<channel id="volumeDimmer" typeId="volumeChannelDimmer" />
-			<channel id="volumeDb" typeId="volumeChannelDb" />
-			<channel id="mute" typeId="muteChannel" />
-			<channel id="setInputSource" typeId="setInputSourceChannel" />
-			<channel id="displayInformation" typeId="displayInformationChannel" />
-		</channels>
+		<channel-groups>
+			<channel-group typeId="displayInformationGroup" id="displayInformation" />
+            <channel-group typeId="zone1Controls" id="zone1" />
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml]: please replace the spaces by tabs

File: addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -96,7 +129,7 @@
 		<state min="0" max="100" pattern="%d %%" />
 	</channel-type>
 
-	<channel-type id="volumeChannelDb" advanced="true">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml]: why did you remove the advanced flag here? Imho there should only be the dimmer channel by default - no need to have two volume controls for everyone.

Comment (Stratehm) [inline addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml]: It should not have been commited. => Revert.

**Comment by ThomDietrich** [comment]
Seems like details altered here need to be reflected in the README.md: https://community.openhab.org/t/pioneer-binding-update-dokumentation/18544

**Comment by Stratehm** [comment]
Indeed. But unfortunately, the behavior of the binding is not as expected. An heavy refactoring of the binding for issues #1206 and #1344 is in progress and configuration will heavily change. I will update the README file with the next PR.

**Comment by ThomDietrich** [comment]
Good to know! Pinging @SpaceMaster85 because of https://github.com/openhab/openhab2-addons/issues/1594

File: addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -82,6 +75,46 @@
...
 	</thing-type>
 
+	<channel-group-type id="displayInformationGroup">
+		<label>Display</label>
+		<channels>
+			<channel id="displayInformation" typeId="displayInformationChannel" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone1Controls">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml]: @Stratehm I only came across this now (and had missed it when reviewing the PR): Why did you actually create 3 group-types for the 3 zones? The idea of the groups is that you only define them once and then reuse the same type for the 3 zones. Would it be possible to refactor that binding or is there any reason against doing so?
